frontend/src/config/mozambique/layers.json

```diff
@@ -2954,5 +2954,46 @@
     "start_date": "2020-12-01",
     "end_date": "2021-07-31",
     "opacity": 0.7
+  },
+  "google_flood_gauges": {
+    "title": "Google Flood Gauges",
+    "type": "point_data",
+    "hex_display": false,
+    "data": "http://0.0.0.0:80/google-floods/gauges/?region_code=MZ",
```

Comment:
Note: need a better development set up

Reply:
@ericboucher have you all deployed the prism-api out in surge-like deployments before for testing?